README.md

```diff
@@ -57,39 +57,24 @@ If you have not installed `pre-commit` then please do so [here](https://pre-comm
 
 ## Run Cucumber tests
 
-Below runs by using the AWS stub, with the following defaults:
+Below runs and uses the core stub as default
 
-- DEFAULT_CLIENT_ID="ipv-core-stub-aws-build"
-- ENVIRONMENT=DEV
-- CRI_DEV=common-lambda-dev
+`STACK_NAME=di-ipv-cri-common-api-your-stack-name ENVIRONMENT=dev API_GATEWAY_ID_PRIVATE=xxxx IPV_CORE_STUB_BASIC_AUTH_USER=xxxx IPV_CORE_STUB_BASIC_AUTH_PASSWORD=xxxx IPV_CORE_STUB_URL="https://cri.core.stubs.account.gov.uk" gradle integration-tests:cucumber
```

Comment:
Looks like you are about to get rid of vital information, I suppose maybe it shouldn't live here. we should discuss maybe the wording and usefulness. My gut is you have found this useful and want to stream the content

Maybe remove changes to the readme for now, or add to it but I think we should remove from it at least not until may we run through it together

Reply:
@liamStockwellGDS please can you look into this?